tox.ini

```diff
@@ -1,4 +1,9 @@
 [tox]
 envlist=py27,py34,py35,py36
 [testenv]
-commands=python setup.py test {posargs}
\ No newline at end of file
+commands=
+    flake8 . \
+    --exclude=datadotworld/client/_swagger/,./.tox/*,./.eggs/*,./build/* \
+    --ignore=F401,E121,F403,F405,F841
```

Comment:
Instead of ignoring these warnings globally, I would rather address each instance in code with `# noqa`